bootstrap/terraform/main.tf

```diff
@@ -55,6 +55,10 @@ locals {
 
   argocd_namespace = "argocd"
 
+  # !NOTE!: only enable one AWS provider at a time
```

Comment:
What happens if you enable both? Crossplane doesn't care as long as names do not collide.

Reply:
you get too many CRDs, I think it is ~1400 and cluster becomes slow